dotCMS/src/main/webapp/html/portlet/ext/contentlet/edit_contentlet.jsp

```diff
@@ -34,6 +34,7 @@
 <!DOCTYPE html>
 <script type='text/javascript' src='/dwr/interface/LanguageAjax.js'></script>
 
+<link rel="stylesheet" href="/html/css/dotcms-block-editor.css" />
```

Comment:
Can we add this only when the block editor is present in the form?

Reply:
We can add where the block editor is used, but since we are iterating over that file, it means we are going to duplicate the CSS for each `dotcms-block-editor` present on a form.